### ISSUE TITLE
voice overlay using & gives unexpected audio playback

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#389]
![thing1234](https://user-images.githubusercontent.com/2993720/81197009-c6ff4400-8f8d-11ea-99dc-63592780a9a0.jpg)

ABC code:
```
M: 4/4
L: 1/8
%%text visual correct; audio incorrect
C8 | D8 |
G8 & E8 | A8 & F8 |
%%text rather, above example sounds more like this:
[CE]8 | [DF]8 |
G8 | A8 |
```

This seems to have something to do with how ampersands interact with score linebreaks. Example seems to play correctly when the four measures are on one line.

**Comment by paulrosen** [comment]
Thanks for the report! I'll get to this soon.

**Comment by paulrosen** [comment]
fixed in 6.0.0-beta.10
